src/CLR/Core/CLR_RT_Interop.cpp

```
@@ -978,7 +978,7 @@ signed __int64 &Interop_Marshal_GetField_double(CLR_RT_HeapBlock *pThis, unsigne
 #endif
 UNSUPPORTED_TYPE &Interop_Marshal_GetField_UNSUPPORTED_TYPE(CLR_RT_HeapBlock *pThis, unsigned int fieldIndex)
 {
-    return (UNSUPPORTED_TYPE &)(*((UNSUPPORTED_TYPE *)NULL));
+    return (UNSUPPORTED_TYPE &)(*((UNSUPPORTED_TYPE *)nullptr));
```

Cppcheck (2.10-2)

[error] Null pointer dereference

(nullPointer)